src/utils/helpers.ts

```diff
@@ -476,10 +476,31 @@ const helpers = {
    * @inner
    */
   createNFTOutput(data: string): Output {
+    return this.createDataScriptOutput(data);
+  },
+
+  /**
+   * Create an output with data script
+   *
+   * @memberof Helpers
+   * @inner
+   */
+  createDataScriptOutput(data: string): Output {
     const scriptData = new ScriptData(data);
     // Value 1 and token HTR
     return new Output(1, scriptData.createScript());
   },
+
+  /**
+   * From the base58 of an address we get the type of it, i.e. 'p2pkh' or 'p2sh'
```

Comment:
Could this method return the type `data` as well, or only `p2pkh` and `p2sh`?

Reply:
Actually not. The `data` output does not have an address.